### PR TITLE
FOUR-2549: It is not possible select a value of select list if there is a record list in the form

### PR DESCRIPTION
**Pull request by danloa**
Fixes [https://processmaker.atlassian.net/browse/FOUR-2549](https://processmaker.atlassian.net/browse/FOUR-2549)

Modify page index just if it is not empty

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-2549-processmaker.ci.processmaker.net

**Comment by sanjacornelius** [comment]
This PR also fixes Jira issue [FOUR-2553](https://processmaker.atlassian.net/browse/FOUR-2553)

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-2549-processmaker.ci.processmaker.net

**Review by caleeli — APPROVED** [review]

File: ProcessMaker/ScreenConsolidator.php [hunk 1]
```diff
@@ -119,7 +119,9 @@ private function setNestedScreen($item, &$new, $index0 = 0)
     private function setRecordList($item, &$new, $index0 = 0)
     {
         $pageId = $item['config']['form'];
-        $item['config']['form'] = $pageId + $index0;
+        if ($pageId)  {
```
Comment (caleeli) [inline ProcessMaker/ScreenConsolidator.php:122]: Please keep the string type when solving the PR:
https://github.com/ProcessMaker/processmaker/pull/3613/files

**Comment by caleeli** [comment]
@danloa @nolanpro when solving the merge with PR (https://github.com/ProcessMaker/processmaker/pull/3613) please keep the (string) conversion

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://develop-processmaker.ci.processmaker.net

**Pull request merged**